pkg/ingester-rf1/stream.go

```diff
@@ -246,14 +247,18 @@ func (s *stream) validateEntries(ctx context.Context, entries []logproto.Entry,
 		}
 
 		lineBytes := len(entries[i].Line)
+		metadataBytes := 0
+		for _, metadata := range entries[i].StructuredMetadata {
+			metadataBytes += len(metadata.Name) + len(metadata.Value)
+		}
 		totalBytes += lineBytes
 
 		now := time.Now()
 		if !rateLimitWholeStream && !s.limiter.AllowN(now, len(entries[i].Line)) {
```

Comment:
I think we should accopunt for structured metadata here as well.

Reply:
I will check it

Reply:
done